src/textual/css/styles.py

```diff
@@ -372,6 +372,13 @@ def __textual_animation__(
             # If destination is a number, we can convert that to a scalar
             if isinstance(value, (int, float)):
                 value = Scalar(value, Unit.CELLS, Unit.CELLS)
+            # If destination is a pair of numbers, we can convert it to a
+            # scalar offset
+            elif isinstance(value, tuple) and (
+                list(map(type, value)) == [int, int]
```

Comment:
This basically checks that `value` is of type `tuple[int, int]` or `tuple[float, float]`.
This is the best way I found to implement it.
If anyone has any suggestions, please let me know.